src/store.ts

```diff
@@ -269,7 +271,18 @@ export default class Store {
    * Gets the current election definition.
    */
   public async getElection(): Promise<Election | undefined> {
-    return await this.getConfig(ConfigKey.Election)
+    const election: Election | undefined = await this.getConfig(
+      ConfigKey.Election
+    )
+
+    if (election) {
+      return {
+        markThresholds: { marginal: 0.17, definite: 0.25 },
```

Comment:
hard-wiring these, or maybe pull them from a config file?

Reply:
Should probably be a config file at some point. These are just the defaults in case the election doesn't define them.